discretisedfield/field.py

```diff
@@ -1075,8 +1118,10 @@ def _check_same_mesh_and_field_dim(self, other, ignore_scalar=False):
             )
 
     def _apply_operator(self, other, function, operator):
+        valid = self.valid.copy()
```

Comment:
We do not need a copy here because a copy is automatically created during initialisation of the new field.

```suggestion
        valid = self.valid
```